packages/edit-site/src/components/save-panel/style.scss

```diff
@@ -2,4 +2,51 @@
 	@include break-small() {
 		width: 600px;
 	}
+
+	.components-modal__content {
+		padding-bottom: 0;
+	}
+
+	.entities-saved-states__panel {
+		display: flex;
+		flex-direction: column;
+
+		.components-panel__body {
+			border: 1px solid $gray-200;
+			border-radius: $radius-block-ui;
+			margin-bottom: $grid-unit-20;
+		}
+	}
+
+	.entities-saved-states__panel-header {
+		order: 1;
```

Comment:
TIL about `order`. This seems to create some [accessibility issues](https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns) though.


I think we can just make the change [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/entities-saved-states/index.js#L174), to render the controls after the `sortedPartitionedSavables`.

Reply:
> I think we can just make the change [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/entities-saved-states/index.js#L174), to render the controls after the sortedPartitionedSavables.

Won't that also affect the save panel in edit view?

Reply:
Hm.. you're right yes.